### PR TITLE
Update to 1.2

### PR DESCRIPTION
**Pull request by kvetinac97**
Use with my PR to SynapseAPI, only thing that does not work is sending packets from Server -> Nemisys -> Player. Thats why communication stops after ResourcePacksInfoPacket.

### CODE REVIEW

File: src/main/java/org/itxtech/nemisys/network/RakNetInterface.java [hunk 1]
```diff
@@ -111,12 +111,12 @@ public void emergencyShutdown() {
...
     @Override
     public void openSession(String identifier, String address, int port, long clientID) {
-        PlayerCreationEvent ev = new PlayerCreationEvent(this, Player.class, Player.class, null, address, port);
+        PlayerCreationEvent ev = new PlayerCreationEvent(this, Player.class, Player.class, clientID, address, port);
         this.server.getPluginManager().callEvent(ev);
         Class<? extends Player> clazz = ev.getPlayerClass();
 
         try {
-            Constructor constructor = clazz.getConstructor(SourceInterface.class, Long.class, String.class, int.class);
+            Constructor constructor = clazz.getConstructor(SourceInterface.class, long.class, String.class, int.class);
```
Comment (ghost) [inline src/main/java/org/itxtech/nemisys/network/RakNetInterface.java:119]: long.class, hm? it's not Long?

Comment (kvetinac97) [reply src/main/java/org/itxtech/nemisys/network/RakNetInterface.java:119]: This is not causing the error

Comment (SupremeMortal) [reply src/main/java/org/itxtech/nemisys/network/RakNetInterface.java:119]: If it was the case then `int.class` would be `Integer.class`

File: src/main/java/org/itxtech/nemisys/Player.java [hunk 2]
```diff
@@ -221,6 +221,7 @@ public void sendDataPacket(DataPacket pk, boolean direct) {
     }
 
     public void sendDataPacket(DataPacket pk, boolean direct, boolean needACK) {
+        System.out.println("Sending data packet to player with id "+pk.pid());
```
Comment (Creeperface01) [inline src/main/java/org/itxtech/nemisys/Player.java:224]: remove?

**Comment by kvetinac97** [comment]
Its somehow corrupted....  I give up, someone, please resolve this

**Comment by PeratX** [comment]
Please be patient, we will update it.

**Pull request merged**